### DISCUSSION TITLE
Recommendations from the community

### DISCUSSION THREAD

**Discussion by time-bomb-timmy** [KelvinTegelaar/CIPP#1313] [General]
Can any community members recommend GitHub and Azure 'subscription' levels?
Sorry if I offended you Kelvin.
Thanks in advance.
Tim

**Comment by BNWEIN** [comment]
A free GitHub account is sufficient. 

Not sure I understand the question about Azure subscription?

An Azure subscription is a logical container used to provision resources in Azure. It holds the details of all your resources like virtual machines (VMs), databases, and more. When you create an Azure resource like a VM, you identify the subscription it belongs to.

The subscription it’s self has no cost. You simply pay for the resources you use inside the subscription. The cost to run CIPP (rough estimates) in Azure are mentioned in the FAQ on the official websites.

With questions like these, my advice would be to take the $99 sponsor and let Kelvin host it for you.

**Comment by time-bomb-timmy** [comment]
If I go to Azure and check Subscriptions I have none. There are many offers to choose from including 'Pay as you Go' which sounds like what most people have. I understand billing requirements change so it will vary.
Do others have 'Pay As You Go' Azure subscriptions? Or recommend 'Azure in Open'?

While I like the idea of having someone host for us and supporting the developer, I am not in a position to make those decisions.
Thanks @BNWEIN 
Tim

**Comment by BNWEIN** [comment]
You will need to create a subscription first. If you are a Microsoft Partner you are given a sponsored subscription, and if you aren’t, then pay as you go will be fine. 

The point about sponsoring the project and hosted was more about the requirements and technical ability to set it up, and manage it going forward. You’ll find the cost of running it to be around $35-$50 a month in azure. That on top of your time to manage and maintain it, you’d probably be better of with the hosting option where they take care of the heavy lifting for you.

**Comment by time-bomb-timmy** [comment]
May I ask how much maintenance (hours per month) one could expect to be needed?
Thanks @BNWEIN I appreciate your time and effort.
Tim

**Reply by BNWEIN** [reply]
Well there are regular updates that should be done and token refreshes to keep things secure. 

The updates require you to synch GitHub repositories. 

None of these things are hugely difficult, but based off some of the questions you’ve asked you may just find the hosted option much easier to manage (and I mean that with respect).
